src/pymortests/algorithms/tr.py

```diff
@@ -0,0 +1,30 @@
+# This file is part of the pyMOR project (https://www.pymor.org).
+# Copyright pyMOR developers and contributors. All rights reserved.
+# License: BSD 2-Clause License (https://opensource.org/licenses/BSD-2-Clause)
+
+import pytest
+import numpy as np
+
+from pymor.algorithms.tr import trust_region
+from pymor.core.exceptions import TRError
+from pymordemos.linear_optimization import create_fom
+from pymor.reductors.coercive import CoerciveRBReductor
+from pymor.parameters.functionals import MinThetaParameterFunctional
+
+
+def test_tr():
+    fom, mu_bar = create_fom(10)
+    parameter_space = fom.parameters.space(0, np.pi)
+    initial_guess = fom.parameters.parse([0.25, 0.5])
+    coercivity_estimator = MinThetaParameterFunctional(fom.operator.coefficients, mu_bar)
+    reductor = CoerciveRBReductor(fom, product=fom.energy_product, coercivity_estimator=coercivity_estimator)
+
+    # successful run
+    _, data = trust_region(parameter_space, reductor, radius=.1, initial_guess=initial_guess)
+    assert len(data['mus']) == 22
```

Comment:
This would be a very disappointing number of outer iterations. Instead, setting the default for `atol` in the TR to `1e-16` gives only 4 outer iterations. It is very typical for such TR algorithms that they are very sensitive to tolerance choices.

Reply:
Right, playing around with the values a bit I noticed that choosing `atol <= 1e-6` did not significantly change the convergence behaviour of the TR for this example. Taking a page from `newton.py`s book, I'll use the `atol = 0.` from there.